tests/test_synthesis.py

```diff
@@ -17,6 +17,7 @@
 def test_register():
     """Test basic plugin registration"""
 
+    from basin3d.synthesis import register, get_timeseries_data
```

Comment:
why is this needed if no other changes to the test?

Reply:
register is used below but get_timeseries_data is not.  Good catch